edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Service/Program.cs

```diff
@@ -159,6 +159,7 @@ public static async Task<int> MainAsync(IConfiguration configuration)
                 Option<X509Certificate2> manifestTrustBundle = Option.None<X509Certificate2>();
                 int edgeletTimeoutSecs = configuration.GetValue(Constants.ManagementApiTimeoutSecs, 300);
                 TimeSpan edgeletTimeout = TimeSpan.FromSeconds(edgeletTimeoutSecs);
+                var enableExistingIdentityCleanup = configuration.GetValue("EnableExistingIdentityCleanup", false);
```

Comment:
@micahl - can you check if the env var name looks okay?

Reply:
@nyanzebra / @varunpuranik suggest EnableOrphanedIdentityCleanup for greater specificity